io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/reader/internal/CSVUtil.java

```diff
@@ -33,18 +34,33 @@
  */
 public class CSVUtil implements CSVConstants {
 
+//	/**
+//	  Reads only the first line of a given CSV
+//	 * file
+//	 * 
+//	 * @param provider to get the parameters from
+//	 * @return a reader containing the first line of the CSV file
+//	 * @throws IOException if an I/O operation fails
+//	 */
+	public static CSVReader readFirst(ImportProvider provider) throws IOException {
+
+		return readFirst(provider, 0);
+	}
+
 	/**
-	 * Reads only the first line of a given CSV file
-	 * 
-	 * @param provider provider to get the parameters from
-	 * @return a reader containing the first line of the CSV file
-	 * @throws IOException if an I/O operation fails
+	 * @param provider to get the parameters from
```

Comment:
Maybe also add a description for this method?
For me the documentation on what is returned and on `skipLines` sounds contradictive. Are the first "x" lines skipped or retrieved?

Reply:
Solved. My understanding is: the method `readFirst` reads/retrieves always just one line, either the first line after the header or the first after skipLines-lines of the file.

Reply:
Here @stempler meant also to add a description to the method documentation. Like as below:
```
/**
* This method reads/retrieves always just one line, either the first line after the header or the first line after skipping N number of lines from the given file.
* 
* @param provider to get the parameters from
* @param skipLines how many lines have to be skipped
* @return a reader containing the first line after (no. of lines -
*         skipLines) of the CSV file
* IOException
*/
```